lib/kitchen/provisioner/puppet_apply.rb

```diff
@@ -537,7 +541,7 @@ def run_command
             puppet_verbose_flag,
             puppet_debug_flag,
             puppet_logdest_flag,
-            remove_repo
```

Comment:
It is moved to proper place https://github.com/neillturner/kitchen-puppet/pull/112/files#diff-851e4fa01ea842ebe9366ac535ce73b0R423